docs/product/dev-toolbar/setup.mdx

```diff
@@ -0,0 +1,144 @@
+2. Call `window.SentryToolbar.init(initConfig)` to set up a toolbar instance on each page where you want to see the Dev Toolbar.
+    ```html
+    <html>
+    <head>...</head>
+    <body>
+        …
+        <script async src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+        <script>
+        window.SentryToolbar.init({ … });
+        </script>
+    </body>
+    </html>
+    ```
+
+### Unmounting The Toolbar
+
+If you have called `SentryToolbar.init({...})` to render the toolbar, but now want to manually remove or unmount it from the page, you can call the cleanup function that is returned from `init()`. This will unmount all the injected HTML and CSS. Login credentials will not be removed, so you can re-insert the toolbar and still be authenticated.
+```html
+<script defer src="https://browser.sentry-cdn.com/sentry-toolbar/latest/toolbar.min.js"></script>
+<script>
+  const unmountToolbar = window.SentryToolbar.init({ … });
+
+  // sometime later...
+  unmountToolbar();
+</script>
+```
+
+## Implement Feature Flag Adapter
+
+In order to integrate your Feature Flagging platform with the Dev Toolbar, you will need an Adapter that can read flag data from your provider, and also store and retrieve a list of overrides to apply to your local browser session.
+
+We plan to create adapters for OpenFeature and LaunchDarkly soon!
+
+The interface you need to implement for an Adapter is:
+```javascript
+type FlagValue = boolean | string | number | undefined;
+type FlagMap = Record<string, FlagValue>;
+interface FeatureFlagAdapter {
+  /**
+   * All known flag names and their evaluated values.
+   */
+  getFlagMap: () => Promise<FlagMap>;
+
+  /**
+   * Any overridden or manually set flags and values.
+   */
+  getOverrides: () => Promise<FlagMap>;
+
+  /**
+   * Manually set a flag to be a specific value, overriding the evaluated value.
+   */
+  setOverride: (name: string, override: FlagValue) => void;
+
+  /**
+   * A callback to clear all overrides from this browser.
+   */
+  clearOverrides: () => void;
+
+  /**
+   * Deeplink into your external feature-flag provider and find out more about
+   * this specific flag.
+   */
+    urlTemplate?: undefined | ((name: string) => string | URL | undefined);
+    }
+```
+
+[MockFeatureFlagIntegration.tsx](https://github.com/getsentry/sentry-toolbar/blob/301c31d561a81e3fd8ffc9532aad3a60af685916/src/env/demo/MockFeatureFlagIntegration.tsx) is an example adapter to use as a reference.
+
+## Init Configuration Fields
+
+The following fields can be passed into the `.init()` function.
+
+For example, a minimal configuration looks like:
+```javascript
+window.SentryToolbar.init({
+  organizationSlug: ‘acme’,
+  projectIdOrSlug: ‘website’,
+  environment: ‘production’
```

Comment:
more quotes

```suggestion
  organizationSlug: 'acme',
  projectIdOrSlug: 'website',
  environment: 'production'
```

if i spotted these we should check everything again to make sure none others are still hanging around.